generator/internal/rust/templates/crate/src/builder.rs.mustache

```diff
@@ -21,6 +21,7 @@ limitations under the License.
 
 {{#Codec.PerServiceFeatures}}
 #[cfg(feature = "{{Codec.ModuleName}}")]
+#[cfg_attr(docsrs, doc(cfg(feature = "{{Codec.ModuleName}}")))]
```

Comment:
Neat.

Reply:
Except that it was not working. Fixed now, PTAL.